react-native/Fable.Helpers.ReactNative.fs

```diff
@@ -157,722 +359,1335 @@ module Props =
         | PaddingRight of float
         | PaddingTop of float
         | PaddingVertical of float
-        | Position of string
+        | Position of Position
         | Right of float
         | Top of float
         | Width of float
+        | ZIndex of float
         interface IFlexStyle
 
-    [<KeyValueList>]
-    type IViewStyle =
-        interface end
 
     [<KeyValueList>]
     type ViewStyle =
-        // FlexStyle
-        | AlignItems of ItemAlignment
-        | AlignSelf of Alignment
-        | BorderBottomWidth of float
-        | BorderLeftWidth of float
-        | BorderRightWidth of float
-        | BorderTopWidth of float
-        | BorderWidth of float
-        | Bottom of float
-        | Flex of int
-        | FlexDirection of FlexDirection
-        | FlexWrap of string
-        | Height of float
-        | JustifyContent of JustifyContent
-        | Left of float
-        | Margin of float
-        | MarginBottom of float
-        | MarginHorizontal of float
-        | MarginLeft of float
-        | MarginRight of float
-        | MarginTop of float
-        | MarginVertical of float
-        | Padding of float
-        | PaddingBottom of float
-        | PaddingHorizontal of float
-        | PaddingLeft of float
-        | PaddingRight of float
-        | PaddingTop of float
-        | PaddingVertical of float
-        | Position of string
-        | Right of float
-        | Top of float
-        | Width of float
-        // End FlexStyle   
-        // TransformsStyle
-        | Transform of obj * obj * obj * obj * obj * obj * obj * obj * obj * obj * obj * obj
-        | TransformMatrix of ResizeArray<float>
-        | Rotation of float
-        | ScaleX of float
-        | ScaleY of float
-        | TranslateX of float
-        | TranslateY of float
-        // End TransformsStyle 
+        | BackfaceVisibility of string
         | BackgroundColor of string
         | BorderBottomColor of string
         | BorderBottomLeftRadius of float
         | BorderBottomRightRadius of float
+        | BorderBottomWidth of float
         | BorderColor of string
         | BorderLeftColor of string
         | BorderRadius of float
         | BorderRightColor of string
+        | BorderRightWidth of float
+        | BorderStyle of (* TODO StringEnum solid | dotted | dashed *) string
         | BorderTopColor of string
         | BorderTopLeftRadius of float
         | BorderTopRightRadius of float
+        | BorderTopWidth of float
         | Opacity of float
-        | Overflow of string
+        | Overflow of (* TODO StringEnum visible | hidden *) string
```

Comment:
Shall we add these `StringEnum`s (also for BorderStyle) above by hand? Unfortunately, there are many of these all over the place and could be a bit tedious.